frontend/src/components/common/carousel/styles.css

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+  position: relative;
+  height: 100%;
+}
+
+.swiper-slide img {
+  display: block;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.mySwiper2 {
+  width: 538px;
+  height: 376px;
+}
+.mySwiper {
+  height: 100px;
+  width: 538px;
+  box-sizing: border-box;
+  padding: 10px 0;
+  margin-top: 10px;
+}
+
+.mySwiper .swiper-slide {
+  width: 25%;
+  height: 100%;
+  opacity: 0.5;
+}
+
+.mySwiper .swiper-slide-thumb-active {
+  opacity: 1;
+}
+
+.mySwiper .swiper-slide-thumb-active .swiper-underline {
+  position: absolute;
+  bottom: 0;
+  height: 3px;
+  width: 98%;
+  background-color: blue;
+  border-bottom-left-radius: 30%;
+  border-bottom-right-radius: 30%;
+}
+
+.mySwiper2 {
+  border-radius: 10px;
```

Comment:
Do we have default variables for border-radius, padding, etc.?

Reply:
i took almost all variables from figma